addons/website_slides/controllers/main.py

```diff
@@ -735,6 +745,7 @@ def slide_view(self, slide, **kwargs):
         values['signup_allowed'] = request.env['res.users'].sudo()._get_signup_invitation_scope() == 'b2c'
 
         if kwargs.get('fullscreen') == '1':
+            values.update(self._slide_channel_prepare_review_values(slide.channel_id))
```

Comment:
With that new call we add channel that we just poped hereabove. Seems quite strange, why do we re-add something we explicitly removed ?

Reply:
Because it's used only in fullscreen mode, so now I drop the key after this condition